stablehlo/integrations/python/mlir/dialects/vhlo.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+# Copyright 2022 The StableHLO Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# pylint: disable=wildcard-import,relative-beyond-top-level,g-import-not-at-top
+from ._vhlo_ops_gen import *
+from .._mlir_libs._vhlo import *
+
+
+# Backward compatibility with the old way of registering VHLO dialect
```

Comment:
I don't think we need this :)

Reply:
Don't need this file at all?

Reply:
Or just not the function. I'm not sure if having the import is required for something. This file is listed as the "Source" for VHLO python in CMake

```
declare_mlir_dialect_python_bindings(
  ADD_TO_PARENT VhloPythonSources.Dialects
  ROOT_DIR "${CMAKE_CURRENT_SOURCE_DIR}/mlir"
  TD_FILE dialects/VhloOps.td
  SOURCES dialects/vhlo.py
  DIALECT_NAME vhlo)

```

Reply:
We need the file but don't need the function.